### ISSUE TITLE
[<Ray component: Ray Tune, RLlib] 

### ISSUE THREAD

**Issue by ChristosPeridis** [ray-project/ray#37348]
### What happened + What you expected to happen

Hello dear members of the Ray Team,

I have been using RLlib and Ray tune for conducting Reinforcement Learning experiments on a plethora of different custom OpenAI Gym environments. Recently I started using the[ RoboHive API](https://github.com/vikashplus/robohive). RoboHive API uses MuJoCo simulator and mujoco-py as backend for the Python bindings with the simulator. Th FAIR Team has developed a plethora of different custom OpenAI Gym environments to address different robotic scenarios and tasks. Because I am working on a projects that targets the integration of Reinforcement Learning in robotic scenarios I wanted to use the custom Gym environments provided by the RoboHive API in combination with Ray Tune and RLlib to test the behaviour of some of SOTA Reinforcement Learning algorithms (like PPO, SAC, DD-PPO, etc.). I had an issue ( #37231) in the past regarding registering the custom RoboHive environment to the Ray API. I have resolved this issue and know I am facing another issue which is that the system is having a trouble building the Reinforcement Learning environment when a worker is initialized for running PPO algorithm (regardless if I am using 0, 1 or more workers). The error that I am receiving is the following:

INFO:root:running build_ext
(PPO pid=27623) INFO:root:building 'mujoco_py.cymj' extension
(PPO pid=27623) INFO:root:gcc -pthread -B /home/cocp5/anaconda3/envs/rhRL94/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -Wall -fPIC -O2 -isystem /home/cocp5/anaconda3/envs/rhRL94/include -I/home/cocp5/anaconda3/envs/rhRL94/include -fPIC -O2 -isystem /home/cocp5/anaconda3/envs/rhRL94/include -fPIC -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/binaries/linux/mujoco210/include -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/numpy/core/include -I/home/cocp5/anaconda3/envs/rhRL94/include/python3.9 -c /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/cymj.c -o /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/generated/_pyxbld_2.0.2.13_39_linuxcpuextensionbuilder/temp.linux-x86_64-cpython-39/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/cymj.o -fopenmp -w
(PPO pid=27623) INFO:root:gcc -pthread -B /home/cocp5/anaconda3/envs/rhRL94/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -Wall -fPIC -O2 -isystem /home/cocp5/anaconda3/envs/rhRL94/include -I/home/cocp5/anaconda3/envs/rhRL94/include -fPIC -O2 -isystem /home/cocp5/anaconda3/envs/rhRL94/include -fPIC -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/binaries/linux/mujoco210/include -I/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/numpy/core/include -I/home/cocp5/anaconda3/envs/rhRL94/include/python3.9 -c /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/gl/osmesashim.c -o /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/generated/_pyxbld_2.0.2.13_39_linuxcpuextensionbuilder/temp.linux-x86_64-cpython-39/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/gl/osmesashim.o -fopenmp -w
(PPO pid=27623) INFO:root:gcc -pthread -B /home/cocp5/anaconda3/envs/rhRL94/compiler_compat -shared -Wl,-rpath,/home/cocp5/anaconda3/envs/rhRL94/lib -Wl,-rpath-link,/home/cocp5/anaconda3/envs/rhRL94/lib -L/home/cocp5/anaconda3/envs/rhRL94/lib -L/home/cocp5/anaconda3/envs/rhRL94/lib -Wl,-rpath,/home/cocp5/anaconda3/envs/rhRL94/lib -Wl,-rpath-link,/home/cocp5/anaconda3/envs/rhRL94/lib -L/home/cocp5/anaconda3/envs/rhRL94/lib /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/generated/_pyxbld_2.0.2.13_39_linuxcpuextensionbuilder/temp.linux-x86_64-cpython-39/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/cymj.o /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/generated/_pyxbld_2.0.2.13_39_linuxcpuextensionbuilder/temp.linux-x86_64-cpython-39/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/gl/osmesashim.o -L/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/binaries/linux/mujoco210/bin -Wl,--enable-new-dtags,-R/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/binaries/linux/mujoco210/bin -lmujoco210 -lglewosmesa -lOSMesa -lGL -o /home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/generated/_pyxbld_2.0.2.13_39_linuxcpuextensionbuilder/lib.linux-x86_64-cpython-39/mujoco_py/cymj.cpython-39-x86_64-linux-gnu.so -fopenmp
2023-07-12 15:33:05,362	ERROR trial_runner.py:1088 -- Trial PPO_RoboHive_Pick_Place_0_f04a6_00000: Error processing event.
ray.tune.error._TuneNoNextExecutorEventError: Traceback (most recent call last):
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/tune/execution/ray_trial_executor.py", line 1070, in get_next_executor_event
    future_result = ray.get(ready_future)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/_private/worker.py", line 2311, in get
    raise value
ray.exceptions.RayActorError: The actor died because of an error raised in its creation task, ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/__init__.py", line 15, in <module>
    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 499, in <module>
    cymj = load_cython_ext(mujoco_path)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 106, in load_cython_ext
    mod = load_dynamic_ext('cymj', cext_so_path)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 125, in load_dynamic_ext
    return loader.load_module()
ImportError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0

During handling of the above exception, another exception occurred:

ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 441, in __init__
    super().__init__(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/tune/trainable/trainable.py", line 169, in __init__
    self.setup(copy.deepcopy(self.config))
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 566, in setup
    self.workers = WorkerSet(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 169, in __init__
    self._setup(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 259, in _setup
    self._local_worker = self._make_worker(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 941, in _make_worker
    worker = cls(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/rollout_worker.py", line 585, in __init__
    self.env = env_creator(copy.deepcopy(self.env_context))
  File "<ipython-input-7-2cb5b960c0e5>", line 3, in env_creator
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 156, in make
    return registry.make(id, **kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 101, in make
    env = spec.make(**kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 73, in make
    env = cls(**_kwargs)
  File "/home/cocp5/robohive/robohive/envs/arms/pick_place_v0.py", line 41, in __init__
    super().__init__(model_path=model_path, obsd_model_path=obsd_model_path, seed=seed)
  File "/home/cocp5/robohive/robohive/envs/env_base.py", line 57, in __init__
    self.sim = SimScene.get_sim(model_path)
  File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 56, in get_sim
    return SimScene.create(model_handle=model_handle, backend=SimBackend.MUJOCO_PY)
  File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 42, in create
    from robohive.physics import mjpy_sim_scene  # type: ignore
  File "/home/cocp5/robohive/robohive/physics/mjpy_sim_scene.py", line 15, in <module>
    import_utils.mujoco_py_isavailable()
  File "/home/cocp5/robohive/robohive/utils/import_utils.py", line 11, in mujoco_py_isavailable
    raise ModuleNotFoundError(f"{e}. {help}")
ModuleNotFoundError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0. 
        Options:
            (1) follow setup instructions here: https://github.com/openai/mujoco-py/
            (2) install mujoco_py via pip (pip install mujoco_py)
            (3) install free_mujoco_py via pip (pip install free-mujoco-py)

(PPO pid=27623) 2023-07-12 15:33:05,354	ERROR worker.py:763 -- Exception raised in creation task: The actor died because of an error raised in its creation task, ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/__init__.py", line 15, in <module>
(PPO pid=27623)     from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 499, in <module>
(PPO pid=27623)     cymj = load_cython_ext(mujoco_path)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 106, in load_cython_ext
(PPO pid=27623)     mod = load_dynamic_ext('cymj', cext_so_path)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 125, in load_dynamic_ext
(PPO pid=27623)     return loader.load_module()
(PPO pid=27623) ImportError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0
(PPO pid=27623) 
(PPO pid=27623) During handling of the above exception, another exception occurred:
(PPO pid=27623) 
(PPO pid=27623) ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 441, in __init__
(PPO pid=27623)     super().__init__(
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/tune/trainable/trainable.py", line 169, in __init__
(PPO pid=27623)     self.setup(copy.deepcopy(self.config))
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 566, in setup
(PPO pid=27623)     self.workers = WorkerSet(

File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 169, in __init__
(PPO pid=27623)     self._setup(
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 259, in _setup
(PPO pid=27623)     self._local_worker = self._make_worker(
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 941, in _make_worker
(PPO pid=27623)     worker = cls(
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/rollout_worker.py", line 585, in __init__
(PPO pid=27623)     self.env = env_creator(copy.deepcopy(self.env_context))
(PPO pid=27623)   File "<ipython-input-7-2cb5b960c0e5>", line 3, in env_creator
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 156, in make
(PPO pid=27623)     return registry.make(id, **kwargs)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 101, in make
(PPO pid=27623)     env = spec.make(**kwargs)
(PPO pid=27623)   File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 73, in make
(PPO pid=27623)     env = cls(**_kwargs)
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/envs/arms/pick_place_v0.py", line 41, in __init__
(PPO pid=27623)     super().__init__(model_path=model_path, obsd_model_path=obsd_model_path, seed=seed)
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/envs/env_base.py", line 57, in __init__
(PPO pid=27623)     self.sim = SimScene.get_sim(model_path)
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 56, in get_sim
(PPO pid=27623)     return SimScene.create(model_handle=model_handle, backend=SimBackend.MUJOCO_PY)
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 42, in create
(PPO pid=27623)     from robohive.physics import mjpy_sim_scene  # type: ignore
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/physics/mjpy_sim_scene.py", line 15, in <module>
(PPO pid=27623)     import_utils.mujoco_py_isavailable()
(PPO pid=27623)   File "/home/cocp5/robohive/robohive/utils/import_utils.py", line 11, in mujoco_py_isavailable
(PPO pid=27623)     raise ModuleNotFoundError(f"{e}. {help}")
(PPO pid=27623) ModuleNotFoundError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0. 
(PPO pid=27623)         Options:
(PPO pid=27623)             (1) follow setup instructions here: https://github.com/openai/mujoco-py/
(PPO pid=27623)             (2) install mujoco_py via pip (pip install mujoco_py)
(PPO pid=27623)             (3) install free_mujoco_py via pip (pip install free-mujoco-py)
2023-07-12 15:33:05,383	ERROR ray_trial_executor.py:118 -- An exception occurred when trying to stop the Ray actor:Traceback (most recent call last):
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/tune/execution/ray_trial_executor.py", line 109, in _post_stop_cleanup
    ray.get(future, timeout=timeout)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/_private/worker.py", line 2311, in get
    raise value
ray.exceptions.RayActorError: The actor died because of an error raised in its creation task, ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/__init__.py", line 15, in <module>
    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 499, in <module>
    cymj = load_cython_ext(mujoco_path)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 106, in load_cython_ext
    mod = load_dynamic_ext('cymj', cext_so_path)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/mujoco_py/builder.py", line 125, in load_dynamic_ext
    return loader.load_module()
ImportError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0

During handling of the above exception, another exception occurred:

ray::PPO.__init__() (pid=27623, ip=172.21.2.236, repr=PPO)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 441, in __init__
    super().__init__(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/tune/trainable/trainable.py", line 169, in __init__
    self.setup(copy.deepcopy(self.config))
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/algorithms/algorithm.py", line 566, in setup
    self.workers = WorkerSet(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 169, in __init__
    self._setup(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 259, in _setup
    self._local_worker = self._make_worker(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/worker_set.py", line 941, in _make_worker
    worker = cls(
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/ray/rllib/evaluation/rollout_worker.py", line 585, in __init__
    self.env = env_creator(copy.deepcopy(self.env_context))
  File "<ipython-input-7-2cb5b960c0e5>", line 3, in env_creator
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 156, in make
    return registry.make(id, **kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 101, in make
    env = spec.make(**kwargs)
  File "/home/cocp5/anaconda3/envs/rhRL94/lib/python3.9/site-packages/gym/envs/registration.py", line 73, in make
    env = cls(**_kwargs)
  File "/home/cocp5/robohive/robohive/envs/arms/pick_place_v0.py", line 41, in __init__
    super().__init__(model_path=model_path, obsd_model_path=obsd_model_path, seed=seed)
  File "/home/cocp5/robohive/robohive/envs/env_base.py", line 57, in __init__
    self.sim = SimScene.get_sim(model_path)
  File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 56, in get_sim
    return SimScene.create(model_handle=model_handle, backend=SimBackend.MUJOCO_PY)
  File "/home/cocp5/robohive/robohive/physics/sim_scene.py", line 42, in create
    from robohive.physics import mjpy_sim_scene  # type: ignore
  File "/home/cocp5/robohive/robohive/physics/mjpy_sim_scene.py", line 15, in <module>
    import_utils.mujoco_py_isavailable()
  File "/home/cocp5/robohive/robohive/utils/import_utils.py", line 11, in mujoco_py_isavailable
    raise ModuleNotFoundError(f"{e}. {help}")
ModuleNotFoundError: /lib/x86_64-linux-gnu/libLLVM-12.so.1: undefined symbol: ffi_type_sint32, version LIBFFI_BASE_7.0. 
        Options:
            (1) follow setup instructions here: https://github.com/openai/mujoco-py/
            (2) install mujoco_py via pip (pip install mujoco_py)
            (3) install free_mujoco_py via pip (pip install free-mujoco-py)

What is so strange about this issue is the fact that it seems that there is a problem with building the mujoco_py library when running the RoboHive environment through the Ray API, whilst there is no problem when running the RoboHive environment alone or running for example the example command that it is supplied in the main GitHub page of the RoboHive Repository:

python -m robohive.utils.examine_env -e FrankaReachRandom-v0

or

python -m robohive.utils.examine_env -e FrankaPickPlaceFixed-v0

This is why I find it so difficult to understand where the problem originates, and what to do in order to resolve it. Could it be an issue with multiprocessing and how new processes or nodes are created in Ray?

Thank you very much in advance for your valuable help!!!

I am always at your disposal for any further clarification of the setup I am using or the issue that I am facing.

Kind reagrads,

Christos Peridis

### Versions / Dependencies

I am running my code in a jupyter notebook using Jupyter-Lab in an Ubuntu 20.04.06 LTS WSL2 system on Windows11. I am using Ray API version 2.2.0, PyTorch version 2.0.1, TensorFlow version 2.12.0, TensorFlow-Probability version 0.20.1, RoboHive version 0.5.0 and mujoco-py version 2.1.2.14 and MuJoCo simulator version 2.1.0 (210). I will also attach an environment.yml file with all my conda environment dependencies. 
[rhRL94_Ubuntu20_b_Ray220_a_environment.zip](https://github.com/ray-project/ray/files/12030378/rhRL94_Ubuntu20_b_Ray220_a_environment.zip)

### Reproduction script

[RoboHive_Ray_Pick_Place_V0.zip](https://github.com/ray-project/ray/files/12030391/RoboHive_Ray_Pick_Place_V0.zip)
Here I am attaching the notbook that I have created to run my experiment code.

### Issue Severity

High: It blocks me from completing my task.

**Comment by ArturNiederfahrenhorst** [comment]
If you simply write a file with the sole content of
`
from gymnasium.envs.mujoco.mujoco_env import MujocoEnv
`
What happens?

**Comment by ChristosPeridis** [comment]
Hello dear @ArturNiederfahrenhorst ,

Thank you very much for your response. I have managed to overcome this issue by upgrading to the dev version of RoboHive which  uses the newer _mujoco_ bindings instead of the old ones that rely on the buggy _mujoco-py_ library. That managed to elevate the issue. The newer conda environment that I have created uses Ray version 1.6.0 which is the version of Ray that does not create any pip dependency conflicts  with the required packages from RoboHive. I am attaching the environment.yml file that I have extracted from this working conda environment!

[rhRL3906_RoboHive_dev_environment.zip](https://github.com/ray-project/ray/files/12111389/rhRL3906_RoboHive_dev_environment.zip)

I have tested that one can create a new conda environment from the above .yml, to test that no pip errors are caused from the environment configuration. In #37456 I am using the conda environment generated from the above .yml file. 

Thank you very much for the help and the support!!!

Kind regards,

Christos Peridis
